vcpkg/examples/assetcaching-script.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Asset Caching with an alternate storage source
+description: Learn about adding a new storage source
+ms.date: 4/18/2023
+---
+
+# Asset Caching with an alternate storage source
+
+[!INCLUDE [experimental](../../includes/experimental.md)]
+
+Your scenario might need a different storage source such as NuGet feed, Universal Packages, or something else.
```

Comment:
As an continuation of the above point, a How-To shouldn't distract the user with other possibilities.